guidelines/MONAI-Workloads.md

```diff
@@ -0,0 +1,436 @@
+The plain radiology modality workload represents ~50% of the Clinical use cases which are responsible for ~55% of the data volume processed (using assumptions in Modality Imaging data volume).  The ultrasound modality workload represents ~22% of the Clinical use cases which is responsible for ~5.4% of the data volume processed.  The CT modality workload represents ~15% of the clinical use cases and is responsible for ~21% of the data volume processed. 
+
+Hardware to support AI for a combined workload for the modalities has traditionally varied. In general, IT infrastructure will assess the mix of modalities and clinical workloads based on the use cases to be supported.  Next, the appropriate mix and utilization method based on these clinical workloads will determine what hardware and compute resources are deployed. 
+
+Typical hardware specifications to be used for the computational workloads are provided below based on specifications seen in the marketplace for AI deployments.
+
+
+### Hardware Targets {#hardware-targets}
+| System   | Hardware Specifications for AI Platform (representative)                                                                                                                                                                                                                                                                                                                                                      |
+| -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| System A | 1U, dual socket rack system supports up to: Two Intel Xeon Scalable processors. Twenty four DIMM slots. Storage capacity of four Graphical Processing Units. A100 80GB Two power supply units (PSUs) Optional two 2.5-inch cabled SATA SSDs installed only in PSU 2 bay.                                                                                                                                      |
+| System B | GPU Up to 4x NVIDIA GPUs RTX 30XX, RTX A6000, RTX A5000, RTX A4000, and Quadro RTX options Processor AMD Threadripper or Intel Core i9 Configurable up to 64 cores, 128 threads, and 256 MB cache Memory Up to 1 TB Fits up to eight 128 GB LRDIMMs at 3200 MHz OS drive Up to 2 TB 3,200 MB/s seq. read and 2,000 MB/s seq. write Extra storage Up to 61 TB Fits up to eight 7.68 TB SATA SSDs. Power supply |
+## Appendix {#appendix}
+
+
+### Clinical Workloads Out of Scope  {#clinical-workloads-out-of-scope}
+
+For the purpose of this document, the following workloads are out of scope:
+
+**Digital Pathology** 
+
+Analyze pathology images with AI algorithms.
+
+
+### Image Enhancement {#image-enhancement}
```

Comment:
I think ### should be changed to **.
Image Enhancement is the subset of "Clinical Workloads Out of Scope" with Digital Pathology, Medical Text Analysis, and so on.

Reply:
Correct. Fixed